src/components/FloatingHelper/ClosablePopover/ClosablePopover.spec.tsx

```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
 import { mount } from 'enzyme';
+import defaults = require('lodash/defaults');
 import * as eventually from 'wix-eventually';
-import { createDriverFactory } from 'wix-ui-test-utils/driver-factory';
-import { isTestkitExists } from 'wix-ui-test-utils/vanilla';
-import { isEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { createDriverFactory, DriverFactory, BaseDriver } from 'wix-ui-test-utils/driver-factory';
+import { enzymeTestkitFactoryCreator } from 'wix-ui-test-utils/enzyme';
 import { closablePopoverDriverFactory, ClosablePopoverDriver } from './ClosablePopover.driver';
 import { ClosablePopover, ClosablePopoverProps } from './ClosablePopover';
-import defaults = require('lodash/defaults');
+import { createEnzymeDriverFactory } from '../../../../test/testkitUtils';
 
 describe('ClosablePopover', () => {
   const createDriver = createDriverFactory(closablePopoverDriverFactory);
-  const createComponent = (partialProps?: Partial<ClosablePopoverProps>) => (
+  const createEnzymeDriver = createEnzymeDriverFactory<ClosablePopover, ClosablePopoverDriver>(closablePopoverDriverFactory);
+  const waitForClose = async () => new Promise((res, rej) => setTimeout(res, ClosablePopover.defaultProps.timeout * 2)); // * 2 as arbitrary safety 
```

Comment:
how come the waitForClose in the other test is waiting for the dataHook, and here you just wait for some constant time?

Reply:
What "other test" are you refering to?